pages.es/common/vim.md

```diff
@@ -0,0 +1,37 @@
+# vim
+
+> Vim (Vi IMproved), un editor de texto para la línea de comandos, que proporciona varios modos para diferentes tipos de manipulación de texto.
+> Pulsando `i` entra en el modo insertar. `<Esc>` habilita el uso de comandos de Vim.
+> Más información: <https://www.vim.org>.
+
+- Abre un archivo:
+
+`vim {{ruta/al/archivo}}`
+
+- Ver el manual propio de Vim:
+
+`:help<Enter>`
+
+- Guarda y sale:
+
+`:wq<Enter>`
+
+- Abre un archivo en un número de línea especificado:
+
+`vim +{{número_de_línea}} {{ruta/al/archivo}}`
```

Comment:
This is open to discussion but if there are variables/placeholders shouldn't have accents
```suggestion
`vim +{{numero_de_línea}} {{ruta/al/archivo}}`
```

Reply:
that doesn't like much to me...
You are not writing a number, that is an indication that there the user must replace the text with a number. So I think that should be an accent, to write the word properly (IMHO)

Reply:
We already discussed that in #5924 and the outcome was to keep accents, unless there's a good reason not to.

Reply:
I would go for that option too...